client/elements/text/sc-text-image.html

```diff
@@ -104,23 +116,34 @@
               if (!this.imageData || !this.imageData[0]) {
                 return;
               };
-              let pageNumber = 0;
-              if (this.imageData.some(page => page.pageNumber === this.pageNumber)) {
-                  pageNumber = this.pageNumber - 1;
-              }
               requestAnimationFrame(() => {
                   const options = {
-                    url: (e) => e.dataUrl,
+                    url: `'src'`,
```

Comment:
This line is interesting. How does this work?

Reply:
See readme file with the viewerjs element. It's an interesting component. It takes the src from the image and displays that in the viewer. I have to hide the actual image that is in template itself as display.none.